src/_posts/languages/python/django/2000-01-01-start.md

```diff
@@ -243,10 +247,19 @@ list of domains which will be used to access the app.
 scalingo --app app-name env-set ALLOWED_HOSTS=app-name.osc-fr1.scalingo.io,example.com
 ```
 
+## Deploy your application
+
+Your app is almost done, now we need to ship it. Django requires that severals
+operation are done each time we deploy the app. In this chapter we will see how
+we can do that on Scalingo.
+
+In Python buildpack, there is 3 hooks we can use: `pre_compile`, `post_compil`
+and `post_deploy`, each is triggered at a specific time and will have some use.
```

Comment:
issue: IMO hooks are not code but feature names. I would also keep a link to the documentation


```suggestion
In Python buildpack, there are 3 hooks we can use: pre-compile, post-compile
and [postdeploy]({% post_url platform/app/2000-01-01-postdeploy-hook %}). 
Each is triggered at a specific time and will have some use.
```